Add fusion docs (#3322)
nextflow-io/nextflow: docs/kubernetes.rst

The workflow application has to be containerised using the usual Nextflow :ref:`container<process-container>` directive.

.. tip::

>   The use of :ref:`wave-page` and :ref:`fusion-page` make obsolete the need to use a shared file system and configure

Reviewer: ```suggestion
  When using :ref:`wave-page` and :ref:`fusion-page` there is no need to use a shared file system and configure
```
